db/migrate/20151024094458_generate_tokens_for_existing_users.rb

```diff
@@ -0,0 +1,7 @@
+class GenerateTokensForExistingUsers < ActiveRecord::Migration
+  def change
+    User.all.each do |user|
```

Comment:
When iterating through `all` of your objects, you should use `find_each` which batches the process 1000 at a time.